README.md

```diff
@@ -269,17 +270,27 @@ This plugin creates device plugin endpoints based on the configurations given in
+|--------------------|----------|----------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------|------------------------------------------------------------------------|
+| "resourceName"     | Y        | Endpoint resource name. Should not contain special characters including hyphens and must be unique in the scope of the resource prefix | string                                                | "sriov_net_A"                                                          |
+| "resourcePrefix"   | N        | Endpoint resource prefix name override. Should not contain special characters                                                          | string Default : "intel.com"                          | "yourcompany.com"                                                      |
+| "deviceType"       | N        | Device Type for a resource pool.                                                                                                       | string value of supported types. Default: "netDevice" | Currently supported values: "accelerator", "netDevice", "auxNetDevice" |
+| "allocationPolicy" | N        | Preferred device allocation policy for a resource pool                                                                                 | string value of supported allocation policy. Default: "" | Currently supported values: "", "packed"                                            |
+| "excludeTopology"  | N        | Exclude advertising of device's NUMA topology                                                                                          | bool Default: "false"                                 | "excludeTopology": true                                                |
+| "selectors"        | N        | Either a single device selector map or a list of maps. The list syntax is preferred. The "deviceType" value determines the device selector options.                                                  | json list of objects or json object. Default: null                   | Example: "selectors": [{"vendors": ["8086"],"devices": ["154c"]}]        |
+| "additionalInfo"   | N        | A map of map to add additional information to the pod via environment variables to devices                                             | json object as string Default: null  | Example: "additionalInfo": {"*": {"token": "3e49019f-412f-4f02-824e-4cd195944205"}} |
 
 Note: "resourceName" must be unique only in the scope of a given prefix, including the one specified globally in the CLI params, e.g. "example.com/10G", "acme.com/10G" and "acme.com/40G" are perfectly valid names.
 
+#### Allocation Policy
+
+The "allocationPolicy" value determines which device in a resource pool is allocated. Each policy acts as following:
```

Comment:
That  is , we need to say what happens in case a policy cannot be satisfied for an allocation.

e.g say we have 3 devices to be allocated but only 2 can be allocated from the same "parent"